nautobot/ipam/fields.py

```diff
@@ -1,96 +1,50 @@
 from django.core.exceptions import ValidationError
 from django.db import models
-from netaddr import AddrFormatError, IPNetwork
+from django.utils.datastructures import DictWrapper
+from netaddr import AddrFormatError, IPNetwork, IPAddress
 
-from . import lookups, validators
 from .formfields import IPNetworkFormField
 
 
-class BaseIPField(models.Field):
-    def python_type(self):
-        return IPNetwork
+class VarbinaryIPField(models.BinaryField):
+    """
+    IP network address
+    """
+
+    description = "IP network address"
+
+    def __init__(self, **kwargs):
+        kwargs["max_length"] = 16
+        super().__init__(**kwargs)
+
+    def db_type(self, connection):
+        engine = connection.settings_dict["ENGINE"]
+
+        # Use 'bytea' type for Postgres.
+        if "postgres" in engine:
+            return super().db_type(connection)
 
-    def from_db_value(self, value, expression, connection):
-        return self.to_python(value)
+        # Or 'varbinary' for everyone else.
+        max_length = DictWrapper(self.__dict__, connection.ops.quote_name, "qn_")
+        return "varbinary(%(max_length)s)" % max_length
 
-    def to_python(self, value):
+    def value_to_string(self, obj):
```

Comment:
This is great! Don't we also still need `from_db_value()` and `to_python() methods defined for the representation of the field values in the ORM as we had discussed?

Docs for reference: https://docs.djangoproject.com/en/3.1/howto/custom-model-fields/#converting-values-to-python-objects